modules/jaggeryapps/ml/site/explore/explore.jag

```diff
@@ -498,6 +508,34 @@ $( document ).ready(function() {
     });
 });
 
+// select the active tab based on the feature types
+function selectActiveTab() {
+    if (numericalFeatureNames.length >1 & categoricalFeatureNames.length >0) {
+        drawScatterPlotBase();
+        $('#scatter-plot-link').click();
+	} else if (categoricalFeatureNames > 1) {
+	    drawParallelSetsBase();
+	    $('#parallel-sets-link').click();
+	} else {
+	    drawTrellisChartBase();
+	    $('#trellis-chart-link').click();
+	}
+}
+
+// disable tabs based on the features and show tooltips
+function disableTabs() {
+    if (!(numericalFeatureNames.length >1 & categoricalFeatureNames.length >0)) {
+        $('#scatter-plot-li').attr('class', 'disabled');
+        $('#scatter-plot-li').attr('title', 'Minimum of two numerical features and one categorical feature required to draw a scatter plot');
+        $('#scatter-plot-link').attr('data-toggle', '');
+	}
+	if(categoricalFeatureNames.length < 2) {
+	    $('#parallel-sets-li').attr('class', 'disabled');
+        $('#parallel-sets-li').attr('title', 'Minimum of two categorical features required for parallel sets');
+        $('#parallel-sets-link').attr('data-toggle', '');
+	}
+}
+

```

Comment:
Hi @manoramahp ,
Above code is repeated in both the file AFAIS. If so, can we move it to utils?